src/app/app.tsx

```diff
@@ -5,6 +5,7 @@ import { QueryClientProvider } from '@tanstack/react-query';
 import { PersistGate } from 'redux-persist/integration/react';
 
 import { LeatherQueryProvider } from '@leather.io/query';
+import '@leather.io/ui/styles';
```

Comment:
Is this taking in `ui/dist-web/styles.css` ? 

I wonder if we can then replace `import './index.css';` in here

Reply:
I tried, but no both are needed

Reply:
Yeah me too, and I remembered then it needs them both. Thanks!